### PR TITLE
added comma on l.35 of store.js, change screening table to use fullname

### PR DESCRIPTION
**Pull request by nondescryptid**
sorry for the silly PR... think something happened while I was resolving merge conflicts to merge #55 into master. 
- Added comma to l35 of storejs 
- Changed Screening Table to use FullName

### CODE REVIEW

File: frontend/src/store.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default new Vuex.Store({
       commit(MUTATIONS.SET_STUDENTS, data)
     },
     ...nativeLanguageActions,
-    ...screeningActions
+    ...screeningActions,
```
Comment (thchia) [inline frontend/src/store.js:33]: `nativeLanguageActions` also appears twice, once on Line 27 and once on Line 34. Let's remove one, and keep all the spread actions together:

```js
actions: {
  ...someActions,
  ...otherActions,
  ...evenMoreActions,
  async getAllStudents() { ... },
  async anotherAction() { ... }
}
```

**Review by thchia — APPROVED** [review]
Left one minor comment, but fine to approve this to get it merged ASAP since it's breaking the build.

**Comment by thchia** [comment]
@szktm let's prioritise this PR, so just go ahead and merge this to fix the pipeline. Up to you whether to include the changes in my comment or do them in a separate PR.

File: frontend/src/store.js (hunk 1 above)
Comment (nondescryptid) [reply frontend/src/store.js:33]: okie, made changes in a new commit

**Pull request merged**